scripts/rcS

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Someone might do `git clone` this repository on macOS (like me), and macOS file systems generally are case-insensitive. Therefore, file "rcS" might confuse the file system. The best way is to rename it to something like `rc-startup` (in git repository) and let the script to copy it into `rcS`.